index.js

```diff
@@ -22,7 +22,7 @@ var events = require('cordova-common').events;
 var path = require('path');
 var fs = require('fs-extra');
 var CordovaError = require('cordova-common').CordovaError;
-const { getInstalledPath } = require('get-installed-path');
+const resolve = Q.denodeify(require('resolve'));
```

Comment:
Not an issue but I know there was an initiative to remove the Q dependency and use Native promises. But this is no problem for now.

https://github.com/apache/cordova/issues/7

Reply:
Everything looks good IMO.